apigee_api_catalog.module

```diff
@@ -40,3 +40,18 @@ function apigee_api_catalog_help($route_name, RouteMatchInterface $route_match)
     default:
   }
 }
+
+/**
+ * Implements hook_page_attachments().
+ */
+function apigee_api_catalog_page_attachments(array &$attachments) {
```

Comment:
Could this not be added to the formatter instead of `hook_page_attachments`?

Reply:
OK, got it working now, was not able to get formatter to properly add it earlier, was thinking it was not possible.